packages/tkg-clusterclass-vsphere/bundle/config/upstream/base.yaml

```diff
@@ -598,7 +598,7 @@ spec:
       - op: replace
         path: /spec/template/spec/template
         valueFrom:
-          template: '{{ (index .TKR_DATA .builtin.machineDeployment.version).osImageRef.template }}'
+          template: '/dc0/vm/photon-3-kube- {{- (index .TKR_DATA .builtin.machineDeployment.version).osImageRef.version }}'
```

Comment:
Using this env variable when creating a management cluster should avoid needing to make this change export _ADDITIONAL_MANAGEMENT_COMPONENT_CONFIGURATION_FILE=https://gist.githubusercontent.com/tenczar/b214012420c3e20582f2f61f0611bdf1/raw/cb46bacbca6b7aee61011c7f750cbb0ea6006e9f/tkr-hack.yml

It defines a custom TKR with a template value of /dc0/photon-3-kube-v1.23.8+vmware.2-tkg.2-81d1a7892ad39f017fbaf59f9907cbe7. The benefit of this approach is that we won't need to revert it and people can create custom TKRs to use whichever template version they want.

Reply:
Thank you @tenczar I'll validate this variable, and update PR later.

Reply:
Thank you @tenczar again, the variable validated, PR commit and description updated.